analyzer/runtime/extract.go

```diff
@@ -121,7 +121,11 @@ type NFTKey struct {
 }
 
 type PossibleNFT struct {
-	NewOwner     *apiTypes.Address
+	// NewOwner has the latest owner if .NumTransfers is more than zero. If
+	// the last transfer burned this NFT instance, then it's the empty string.
```

Comment:
Nice, thank you for the docstrings! Can we keep it as a pointer though? Right now, to realize that the `NewOwner` value should be ignored, one needs to check `NumTransfers == 0`, which is far from obvious.

Optional: I also suggest the special value be something other than the empty string, e.g. `"__BURN__"` or `"0"`. It's currently set very implicitly, via uninitialized values, and it's easy to miss it when reading the code. Also when checking the variable value in an `if`, `"__BURN__"` is a lot more informative than `""`.

Reply:
I'm not feeling that special string idea. I'll add a Burned bool field.